### ISSUE TITLE
Accessibility: MAS05A: Keyboard: keyboard focus is not properly visible for the links

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#35]
#A11yMAS
#MAS20
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
**Scenario 1:**
1.Open https://code.visualstudio.com/ in edge.
2.Navigate to the "Follow@code" Tweet button in footer of the page
3.Observe keyboard focus[Bounded rectangular box].
**Note:** Same issue is observed for all links

**Scenario 2:**
1.Open https://code.visualstudio.com/docs/introvideos/overview in edge.
2.Navigate to  "Tweet" button under "IN THIS ARTICLE" above footer of the page.
3.Observe keyboard focus **[Bounded rectangular box].**
**Note:** Same issue observed in below links 
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/runtimes/nodejs
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/introvideos/overview
https://code.visualstudio.com/docs/editor/accessibility
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/runtimes/nodejs
https://code.visualstudio.com/blogs
https://code.visualstudio.com/updates
https://code.visualstudio.com/docs/supporting/faq

**Actual Result:**
Keyboard focus [Bounded rectangular box] is not visible for the "Tweet" button in the page.

**Expected Result:**
keyboard focus  [Bounded rectangular box] should be visible for "Tweet" button.

**User Impact:**
Users with visual and/or motor impairments who rely on keyboard may not be able to identify that the element is focused or not.

**Screenshots:**
[no bounded rectangular box.zip](https://github.com/wiproaccess/vscode-website/files/689123/no.bounded.rectangular.box.zip)

**Comment by waderyan** [comment]
We're using Twitter's code for this. Will be hard to do.

**Comment by chrisdias** [comment]
With narrator turned on, there is a focus box.

![image](https://cloud.githubusercontent.com/assets/1487073/26424391/e63b3f0e-4085-11e7-94ca-174db09ed738.png)

With narrator turned off, the focus rectangle blends in with the blue color of the widget. However, the color of the widget changes to indicate focus.

No focus:
![image](https://cloud.githubusercontent.com/assets/1487073/26424432/0c65d194-4086-11e7-9564-c70136382d52.png)

Focus on "Follow @code":
![image](https://cloud.githubusercontent.com/assets/1487073/26424466/27b6776e-4086-11e7-94a0-8c98affb6c42.png)

For the followers part of the widget, it becomes underlined when it has focus:

![image](https://cloud.githubusercontent.com/assets/1487073/26424479/35385cf4-4086-11e7-8c43-1e3bca68e80a.png)

Given this, I don't think there is anything for us to fix.
